lib/WebService/Async/Onfido.pm

```diff
@@ -695,7 +695,7 @@ sub live_photo_upload {
             $log->errorf('Request %s received %s with full response as %s',
                 $request->uri,
                 $message,
-                $response->as_string("\n"),
+                $response->content,
```

Comment:
Isn't it good to modify the response for `document_upload` as well? Because of the probability of a similar issue for that call.
https://github.com/binary-com/perl-WebService-Async-Onfido/blob/d32bd25ccc89d9ca263e44d47211ad10c5c30f82/lib/WebService/Async/Onfido.pm#L630

Reply:
true, updated.